trade-finance-manager-api/src/v1/rest-mappings/facility.js

```diff
@@ -1,22 +1,27 @@
 const mapFacilitySnapshot = require('./mappings/facilities/mapFacilitySnapshot');
 const mapFacilityTfm = require('./mappings/facilities/mapFacilityTfm');
-const mapGefFacility = require('./mappings/gef-facilities/mapGefFacility');
+const mapGefFacilitySnapshot = require('./mappings/gef-facilities/mapGefFacilitySnapshot');
 const isGefFacility = require('./helpers/isGefFacility');
 
+/**
+ * Maps the facility object from the database to a dynamically updating facility object used across TFM.
```

Comment:
Yes good idea! I think I meant just that it will reflect changes as you make them in amendments, but appreciate the word `dynamically` might have additional meanings that I don't intend it to mean.

Reply:
That's such a good explanation too😍